### DISCUSSION TITLE
［求助］

### DISCUSSION THREAD

**Discussion by Signedinn** [share121/inter-knot#5063] [灌水]
iOS手机端更新之后要让我登录 要不然用不了 有什么头绪吗
![IMG_4751](https://github.com/user-attachments/assets/93a8880b-fcd9-439b-8b40-483cafa8574f)
![IMG_4215](https://github.com/user-attachments/assets/d252654b-bff8-431f-ab4f-601ebf6091c0)

**Comment by cuijiaqi0406** [comment]
进浏览器登录……？你没登录怎么发的帖子啊喂
